doc/plotting.rst

```diff
@@ -359,19 +286,21 @@ discrete colormap:
 .. ipython:: python
 
     @savefig plotting_listed_levels.png width=4in
-    distance.plot(levels=[2, 5, 10, 11])
+    air2d.plot(levels=[0, 12, 18, 30])
 
-Finally, if you are have `Seaborn <http://stanford.edu/~mwaskom/software/seaborn/>`_ installed, you can also specify a `seaborn` color palete or a list of colors as the ``cmap`` argument:
+Finally, if you have `Seaborn <http://stanford.edu/~mwaskom/software/seaborn/>`_ installed, you can also specify a `seaborn` color palete or a list of colors as the ``cmap`` argument:
 
 .. ipython:: python
 
     flatui = ["#9b59b6", "#3498db", "#95a5a6", "#e74c3c", "#34495e", "#2ecc71"]
     @savefig plotting_custom_colors_levels.png width=4in
-    distance.plot(levels=[1, 2, 4, 5, 7], cmap=flatui)
+    air2d.plot(levels=[0, 12, 18, 30], cmap=flatui)
 
 Maps
 ----
 
+TODO - Update this example to use the tutorial data.
```

Comment:
I'm actually ok with having them there- it shows that we're planning to change it soon, and maybe someone will feel inspired to send in a patch.

But removing them is no problem.